lib/src/components/NxSidebarNavigation/NxSidebarNavigation.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { faBars, faQuestionCircle } from '@fortawesome/free-solid-svg-icons';
+
+import NxButton from '../NxButton/NxButton';
+import NxCloseButton from '../NxCloseButton/NxCloseButton';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { Props, SidebarNavLinkProps, propTypes } from './types';
+
+import './NxSidebarNavitation.scss';
+
+const NxSidebarNavLink = function NxSidebarNavLink({ name, href, icon, current }: SidebarNavLinkProps) {
+  const navigationLinkClassnames = classnames('nx-sidebar__item', {
+    'selected': current
+  });
+
+  return (
+    <a href={href} className={navigationLinkClassnames} key={`nav-link-${name}`}>
+      <NxFontAwesomeIcon icon={icon} />
+      <span className="nx-sidebar__link-name">{name}</span>
+    </a>
+  );
+};
+
+const NxSidebarNavigation = function NxSidebarNavigation(props: Props) {
+  const {
+    isOpen,
+    className,
+    onToggleClick,
+    logoImg,
+    logoText,
+    logoLink,
+    links,
+    helpLink,
+    helpText,
+    collapsedReleaseText,
+    expandedReleaseText,
+    attributions
+  } = props;
+
+  const classes = classnames(className, 'nx-sidebar', {
+    'nx-sidebar--open': isOpen,
+    'nx-sidebar--closed': !isOpen
```

Comment:
As described on the Styling Components page in the gallery, we use un-BEMed utility classes for certain common states, notably `open` and `closed` which are applicable here.

Reply:
> As described on the Styling Components page in the gallery, we use un-BEMed utility classes for certain common states, notably `open` and `closed` which are applicable here.

Changed in eafc84f4ef5fc29a4776b1838f1eeb18f18ef294